.github/workflows/bump-deps.yaml

```diff
@@ -0,0 +1,70 @@
+# Description: Run this by visiting https://github.com/brave/brave-core/actions/workflows/bump-deps.yaml
+# and clicking the `Run workflow` button on the top-right.
+# You can also run it from cli by: `gh workflow run bump-deps.yaml -f ref=commit-hash`
+name: Bump DEPS
+on:
+  workflow_dispatch:
+    inputs:
+      dep:
+        description: "Dependency"
+        required: true
+        type: string
+        default: "vendor/web-discovery-project"
+      ref:
+        description: "New ref for dependency"
+        required: false
+        type: string
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  auto-bump:
+    runs-on: ubuntu-latest
+    env:
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: "0"
+
+      - name: auto-bump
+        run: |
+          set -x
```

Comment:
Is this already using
```bash
shopt -s inherit_errexit
set -eEo pipefail
```
?

If not, I'd suggest always using that in bash scripts.

Reply:
Done